index.html

```diff
@@ -70,6 +76,255 @@ <h1>I'm Abdi</h1></div>
+                        </ul>
+                </div>
+                <div class="seeProjectCards">
+                    <a href="#"><button type="button" id="button_linkProj">See Project</button></a>  
+              </div>
+            
+              
+            
+            </div>
+
+            
+            
+            <div class="cards card6">
+                <h3 class="cards-header">Profesional Art <br>Printing Data</h3>
+                <div class="cards_description">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's 
+                        standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                </div>
+              
+                <div class="card_buttons">
+                    <ul>
+                        <li> <a href="#">  <button type="button" id="button_link">html</button></a></li>
+                                        <li>  <a href="#">   <button type="button" id="button_link">bootstrap</button></a>
+                                        </li>
+                                        <li> <a href="#">  <button type="button" id="button_link">Ruby</button></a></li>
+                        </ul>
+                </div>
+                <div class="seeProjectCards">
+                    <a href="#"><button type="button" id="button_linkProj">See Project</button></a>  
+              </div>
+            
+              
+            
+            </div>
+
+      
+            <div class="cards card7">
+                <h3 class="cards-header">Profesional Art <br>Printing Data</h3>
+                <div class="cards_description">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's 
+                        standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                </div>
+              
+                <div class="card_buttons">
+                    <ul>
+                        <li> <a href="#">  <button type="button" id="button_link">html</button></a></li>
+                                        <li>  <a href="#">   <button type="button" id="button_link">bootstrap</button></a>
+                                        </li>
+                                        <li> <a href="#">  <button type="button" id="button_link">Ruby</button></a></li>
+                        </ul>
+                </div>
+                <div class="seeProjectCards">
+                    <a href="#"><button type="button" id="button_linkProj">See Project</button></a>  
+              </div>
+            
+              
+            
+            </div>
+           
+
+
+        </div>
+       
+       
+        
+
+
+
+
+
+
+
+    </section>
+    <div class="about_Me">
```

Comment:
[OPTIONAL] Kindly organised your code in a well orderly manner without empty spaces. Doing this will make your code more readable and proffessional. 
It could be something similar to the image below.

![Screenshot from 2022-09-24 21-10-19](https://user-images.githubusercontent.com/65233860/192118917-ab324525-a36e-4ed2-941e-1135d930d589.png)